packages/sharing-service/test/unitTests/test-sharing.js

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
can we make a pass after all this lands to remove whatever extraneous imports needed to be suppressed?

Reply:
Sure, I'll remove these disable statements for now. The idea was that test files should never add things to `dependencies`, only `devDependencies`, but I haven't found the right setting for that yet. It's probably in here someplace: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md